src/Facebook/InstantArticles/Client/Client.php

```diff
@@ -98,6 +98,17 @@ public function importArticle($article, $takeLive = false)
     }
 
     /**
+     * Removes an article from your Instant Articles library.
+     *
+     * @param string $canonicalURL The canonical URL of the article to get the status for.
+     */
+    public function removeArticle($canonicalURL) {
+        if ($articleID = $this->getArticleIDFromCanonicalURL($canonicalURL)) {
```

Comment:
I think it would be a good idea to add a check identical to [this one](https://github.com/facebook/facebook-instant-articles-sdk-php/blob/master/src/Facebook/InstantArticles/Client/Client.php#L131) and return an `InstantArticleStatus` instance – both for "not found" and "success"

We should also add at type check like [this one](https://github.com/facebook/facebook-instant-articles-sdk-php/blob/master/src/Facebook/InstantArticles/Client/Client.php#L135).